docs/publishing-modules.md

```diff
@@ -24,65 +30,60 @@ Your `tsconfig.json` file must at least specify the following...
 }
 ```
 
-And your `package.json` file should specify the `types` property. You should also specify the `main` property if your module contains runnable Lua code.
+Your `package.json` file should include the following fields:
 
 ```json title=package.json
 {
-  "main": "./dist/index", // points to ./dist/index.lua
-  "types": "./dist/index" // points to ./dist/index.d.ts
+  "name": "foo",
+  "version": "1.0.0",
+  "description": "A description of your package.",
+  "keywords": ["typescript-to-lua", "lua", "tstl"],
+  "homepage": "https://github.com/AUTHOR_NAME/PROJECT_NAME",
+  "bugs": {
+    "url": "https://github.com/AUTHOR_NAME/PROJECT_NAME/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AUTHOR_NAME/PROJECT_NAME.git"
+  },
+  "license": "MIT",
+  "author": "AUTHOR_NAME",
+  "files": ["dist", "LICENSE", "package.json", "README.md"],
```

Comment:
Please put this back to what it was before, like you mention below most of these files are included in the package by default, and you are now hiding the content of dist which is the most important part